Pages/Index.cshtml

```diff
@@ -3,21 +3,26 @@
 @model IndexModel
 @{
     ViewData["Title"] = "Sips and Bites Navigator";
-    var brand = ViewData["Brand"];
-    List<Brewery> breweries = (List<Brewery>) ViewData["Breweries"];
+    var siteBrand = ViewData["Brand"];
+    List<Brewery> breweryList = (List<Brewery>) ViewData["Breweries"];
```

Comment:
Avoids ambiguity and aligns with the naming style of collections.
Updated naming conventions for readability and consistency.